Bottom: 20d753414434d5cf4d7ed94b3dbe77a9fe1c3a19
Top:    00713460173da27a8d641d2bf51d5e5afd0e1ff3
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-05 13:48:16 -0500

Update docker-entrypoint to call parent container's entrypoint

After updating the parent image, our entrypoint wasn't setting some environment
variables correctly. Using the parent's entrypoint script after our setup
fixes this problem.


---
diff --git a/docker/docker-mediawiki/docker-entrypoint.sh b/docker/docker-mediawiki/docker-entrypoint.sh
index 0e8fc7e2..ff93cb5c 100755
--- a/docker/docker-mediawiki/docker-entrypoint.sh
+++ b/docker/docker-mediawiki/docker-entrypoint.sh
@@ -108,10 +108,8 @@ case ${1} in
     wiki)
         # verify permissions
         chown -R www-data:www-data ${WIKI_BASE_DIR}
-        # Apache gets grumpy about PID files pre-existing
-        rm -f /var/run/apache2/apache2.pid
         # Start apache
-        exec apache2 -D FOREGROUND
+        exec apache2-foreground
         ;;
 
     LocalSettings)
